server/plugin/api_test.go

```diff
@@ -712,6 +712,17 @@ func TestPlugin_handlePickerSelection(t *testing.T) {
 	}
 }
 
+func getHandleDateTimeSelectionRequestBody(date, time, dialogType string) model.SubmitDialogRequest {
```

Comment:
```suggestion
func getHandleDateTimeSelectionRequestBody(date, time, dialogType string) *model.SubmitDialogRequest {
```
Shouldn't it be like this?